mysql-test/suite/galera/t/mdev-30013.test

```diff
@@ -0,0 +1,15 @@
+--source include/galera_cluster.inc
+
+if (!$HA_ARCHIVE_SO) {
+  skip Needs Archive loadable plugin;
+}
+
+INSTALL PLUGIN ARCHIVE SONAME 'ha_archive.so';
```

Comment:
This should be fixed to work on Windows.

See many similar examples in the `archive` test suite, e.g. https://github.com/MariaDB/server/blob/11.1/mysql-test/suite/archive/archive_plugin.test#L7-L8

Reply:
Galera does not support Windows and these tests are not run on Windows. Therefore, I do not see point on this.